src/browsergym/workarena/tasks/base.py

```diff
@@ -64,6 +64,17 @@ def __init__(
         self.timeout = 10000  # ms
 
         self.instance = instance if instance is not None else SNowInstance()
+
+        # Check if workarena-install is done correctly
+        property_name = "workarena.installation.date"
```

Comment:
Minor: no need to define this variable, you could just use it in the function call directly.

Reply:
Sounds good.